dev/loadgen/configs/prod-benchmark-pvc.yaml

```diff
@@ -70,18 +73,18 @@ repos:
   - cloneURL: https://github.com/gitpod-io/template-typescript-node
     cloneTarget: master
     score: 20
-    workspaceImage: eu.gcr.io/gitpod-dev/workspace-images:dd3075638ccf424374cc9c681b3d2338908d07115b28e54856cd45604c33768c
+    workspaceImage: registry.hub.docker.com/gitpod/workspace-full:latest
```

Comment:
@Furisto something seems off here, but I may be mistaken.

On the left is an image that was built by image builder, and persisted to our internal GCR

On the right is an image that was built by dazzle, and persisted to public docker hub, but would still need to be built by image builder to work in a workspace, relying on this could make loadgen "take longer"

@aledbf can you help clarify what is expected/proper here?